static/index.css

```diff
@@ -32,17 +32,38 @@ body {
   width: 100vw;
   height: 100vh;
 
-  background-color: var(--color-error-default);
+  background-color: #d03242;
```

Comment:
As a general rule a static hex value should be a red flag. We should be using the design token css color variables available

```suggestion
  background-color: var(--color-error-default);
```

Reply:
Yes, seen that little late, will fix it

Reply:
You should use the error default red color @aleksandar-mihajlovic

Reply:
It is fixed now